base_comment_template/migrations/13.0.2.0.0/pre-migration.py

```diff
@@ -0,0 +1,12 @@
+# Copyright 2021 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openupgradelib import openupgrade  # pylint: disable=W7936
+
+field_renames = [
+    ("base.comment.template", "base.comment.template", "priority", "sequence"),
```

Comment:
Take care with the proper arguments, that these things are very complicated to be detected, and they are not tested in CI:
```suggestion
    ("base.comment.template", "base_comment_template", "priority", "sequence"),
```